pkgs/tools/compression/zfp/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
     homepage = "https://computing.llnl.gov/projects/zfp";
     description = "Library for random-access compression of floating-point arrays";
     license = licenses.bsd3;
-    maintainers = [ maintainers.spease ];
+    maintainers = with maintainers; [ spease atila ];
     # 64-bit only
     platforms = platforms.aarch64 ++ platforms.x86_64;
```

Comment:
```suggestion
    platforms = platforms.aarch64 ++ platforms.x86_64;
    # error: '_Float16' was not declared in this scope; did you mean '_Float64'?
    broken = stdenv.isAarch64;
```

Reply:
Thanks for the review! Fixed! But darwin_x86 is still failling.

Reply:
I would suggest to mark darwin broken.